src/components/ConnectionSidebar/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { EyeOpenIcon, KeyboardIcon, BackpackIcon } from '@radix-ui/react-icons';
+import React from 'react';
+import { useConnectionsProvider } from '../../contexts/ConnectionsProvider';
+import { Box } from '../base';
+import { SidebarGroup, SidebarItem } from './styles';
+
+export function ConnectionSidebar() {
+  const { selectedConnection } = useConnectionsProvider();
+
+  if (!selectedConnection) {
+    return null;
+  }
+
+  return (
+    <Box
```

Comment:
might be cool to add like flex variants of the box, we can make it clear and avoid css props repetition, like:
```ts
variants: {
  flex: {
    column: { display: "flex'" flexDirection: "column" },
    row: { display: "flex'" flexDirection: "row" },
  }
}
```

Reply:
I like the idea of maintaining a fully abstract Box component, while we can create a Flex component with variants

to use like:

```tsx
<Flex direction="column" align="center">

</Flex>
```